diff --git a/nix/desktop.nix b/nix/desktop.nix
index 3e9c64fbd9ad..c254e4c8a3dd 100644
--- a/nix/desktop.nix
+++ b/nix/desktop.nix
@@ -14,7 +14,7 @@ let

   electronHeaders = pkgs.fetchurl {
     url = "https://artifacts.electronjs.org/headers/dist/v${electron.version}/node-v${electron.version}-headers.tar.gz";
-    sha256 = "sha256-zi/QMwRZ0+FwE9XTE+DiSIeJXAwxmLKEaBWD5W3pMOI=";
+    sha256 = "sha256-zOl8rx6woWh7aeRUOlkTMviKc/EAQQX6nr/MxAx1ZPI=";
   };

   # node-pty ships no Electron-tagged prebuild we can trust to match this
