diff --git a/lollms/paths.py b/lollms/paths.py
index 55fab15..7645289 100644
--- a/lollms/paths.py
+++ b/lollms/paths.py
@@ -124,9 +124,9 @@ class LollmsPaths:
                 "lollms_personal_path":str(Path.home()/"Documents/lollms")
             })
 
-            cfg.lollms_personal_path = input(f"Folder path: ({cfg.lollms_personal_path}):")
-            if cfg.lollms_personal_path=="":
-                cfg.lollms_personal_path = str(Path.home()/"Documents/lollms")
+            # cfg.lollms_personal_path = input(f"Folder path: ({cfg.lollms_personal_path}):")
+            # if cfg.lollms_personal_path=="":
+            cfg.lollms_personal_path = str(Path.home()/"Documents/lollms")
 
             print(f"Selected: {cfg.lollms_personal_path}")
             pp= Path(cfg.lollms_personal_path)
